portal/src/main/java/com/liferay/faces/portal/component/inputsearch/internal/InputSearchRenderer.java

```diff
@@ -241,7 +239,10 @@ protected void dynamicallyAddChildComponents(InputSearch inputSearch) {
 			if (!showButton && (hasButtonAjaxBehavior || (action != null) || (actionListener != null))) {
 				logger.error("Set showButton=\"true\" when using action/actionListener or f:ajax.");
 			}
+
+			facesContext.getViewRoot().markInitialState();
```

Comment:
Was there a call to markInitialState() in your original pull request? I don't remember seeing that.